src/comments/popup.js

```diff
@@ -0,0 +1,99 @@
+import { lookupShow, getComments, postCommentToApi } from '../api/api.js';
+
+const addItemToList = (ul, arr) => {
+  arr.forEach((el) => {
+    ul.innerHTML += `<li class="comment-list-item my-1">${el.creation_date} ${el.username}: ${el.comment}</li>`;
+  });
+};
+
+export const getCounts = (arr) => arr.length;
+
+const incrementCount = (title, commentsInfo) => {
+  const value = getCounts(commentsInfo) + 1;
+  title.textContent = `Comments (${value})`;
+};
+
+export default async function gotoCommentPage(showId) {
```

Comment:
- [ ] Another reminder that the function above  should be an arrow function while following the ES6 rules.

Reply:
We replaced function with an arrow function while following the ES6 rules in popup file.